builder/package.centos-7

```diff
@@ -0,0 +1,46 @@
+if [[ ! -d /tmp/output/centos-7 ]]; then
+  mkdir -p /tmp/output/centos-7
+fi
+
+fpm \
+  -s dir \
+  -t rpm \
+  -v 1 \
+  -n R-${R_VERSION} \
+  --vendor "R Project" \
+  --deb-priority "optional" \
+  --deb-field 'Bugs: https://github.com/rstudio/r-builds/issues' \
+  --url "http://www.r-project.org/" \
+  --description "GNU R statistical computation and graphics system" \
+  --maintainer "RStudio Inc https://github.com/rstudio/r-builds" \
+  --license "GPL-2" \
+  -p /tmp/output/centos-7/ \
+  -d bzip2-devel \
+  -d gcc \
+  -d gcc-c++ \
+  -d gcc-gfortran \
+  -d libcurl-devel \
+  -d libicu-devel \
+  -d libSM \
+  -d libtiff \
+  -d libXmu \
+  -d libXt \
+  -d make \
+  -d openblas-devel \
+  -d pango \
+  -d pcre-devel \
+  -d pcre2-devel \
+  -d perl-Digest-MD5 \
+  -d sudo \
```

Comment:
I think we should remove sudo, and maybe vim/wget. The latter two could be useful, but we didn't add them for the Ubuntu package.

Reply:
I initially tried without vim and wget, but I was getting a missing dependency error when the resulting package was installed that I struggled to diagnose. I ended up putting them back in for this reason. I'll do some more investigation on that, I'm guessing that one of them pulls in another dependency that we need here, I'm just not sure what it is.

Reply:
Oh interesting, I can take a look as well

Reply:
I tried removing vim/wget and got this error from `sessionInfo()` on R 3.5. Is it the same thing you saw?
```r
> sessionInfo()
Error in system(paste(which, shQuote(names[i])), intern = TRUE, ignore.stderr = TRUE) : 
  error in running command
```

The error message isn't great, but it looks like it's coming from `Sys.which()` not finding the `which` command. I was able to get it working after doing a `yum install which`
```r
> traceback()
5: system(paste(which, shQuote(names[i])), intern = TRUE, ignore.stderr = TRUE)
4: withCallingHandlers(expr, warning = function(w) invokeRestart("muffleWarning"))
3: suppressWarnings(system(paste(which, shQuote(names[i])), intern = TRUE, 
       ignore.stderr = TRUE))
2: Sys.which("uname")
1: sessionInfo()
```

Reply:
Yeah, that's the one - it should be fixed now.

I've removed sudo, vim and wget and added which